CHANGELOG.md

```diff
@@ -5,6 +5,9 @@ This CHANGELOG follows the format listed [here ](https://github.com/sensu-plugin
 
 ## [Unreleased]
 
+### Breaking Changes
```

Comment:
I will fix this up for you bit since you changed the threshold to `0` its no longer a breaking change.

Reply:
Please, fix. Thanks.